src/reference/00-Getting-Started/02-sbt-by-example.md

```diff
@@ -132,11 +132,11 @@ Hello
 [success] Total time: 1 s, completed May 6, 2018 4:10:44 PM
 ```
 
-### Set ThisBuild / scalaVersion from sbt shell
+### Set scalaVersion from sbt shell
 
 ```
-sbt:foo-build> set ThisBuild / scalaVersion := "2.12.7"
-[info] Defining ThisBuild / scalaVersion
+sbt:foo-build> set scalaVersion := "2.12.7"
+[info] Defining *:scalaVersion
```

Comment:
@edwardCao What version of sbt are you using? `ThisBuild / scalaVersion` should work fine.

Reply:
I just checked and somehow my shell is defaulting to 0.13 even though I have 1.3 installed. Is this tutorial only for 1.x? If so, how do I force it to use 1.3 if I have other versions installed?

Reply:
> Is this tutorial only for 1.x?

Yes.

> If so, how do I force it to use 1.3 if I have other versions installed?

You could try `sbt -Dsbt.version=1.3.13`, but generally I'd recommend fixing your path situation since we do make changes to sbt Bash/Batch scripts.